src/views/Playgrounds/PlaygroundChannelEditCreate.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { useRef, useState, useEffect } from 'react'
+import { useForm, Controller } from 'react-hook-form'
+
+import { ImageCropDialog, ImageCropDialogImperativeHandle, ViewWrapper } from '@/components'
+import { ChannelCover, FormField, Select, SelectedItem, HeaderTextField, Tooltip } from '@/shared/components'
+
+import { transitions } from '@/shared/theme'
+import {
+  StyledTitleSection,
+  TitleContainer,
+  StyledFormField,
+  StyledTextarea,
+  StyledActionBarTransaction,
+  StyledAvatar,
+  InnerFormContainer,
+} from './PlaygroundChannelEditCreate.style'
+import { Header, SubTitle, SubTitlePlaceholder, TitlePlaceholder } from '../ChannelView/ChannelView.style'
+
+import { requiredValidation, textFieldValidation } from './formValidationOptions'
+import { formatNumberShort } from '@/utils/number'
+
+const languages: SelectedItem[] = [
+  { name: 'English', value: 'english' },
+  { name: 'Polish', value: 'polish' },
+  { name: 'Spanish', value: 'spanish' },
+]
+
+const publicnessItems: SelectedItem[] = [
+  { name: 'Public (Anyone can see this video', value: 'public' },
+  { name: 'Private', value: 'private' },
+]
+
+const channel = {
+  avatar: 'https://picsum.photos/200/300',
+  cover: 'https://eu-central-1.linodeobjects.com/atlas-assets/channel-posters/2.jpg',
+  handle: 'Lorem ipsum dolor sit amet',
+  follows: 1000,
+  description: `Making videos about cars, coffee and travel
+YouTube: SeenThroughGlass
+Podcast: BehindTheGlass
+Business: Lucy.Bayliss@mcsaatchimerlin.com`,
+}
+
+type Inputs = {
+  channelName?: string
+  description?: string
+  selectedPublicness?: string
+  selectedLanguage?: string
+  avatar: string | null
+  cover: string | null
+}
+
+const PlaygroundChannelEdit = () => {
+  const { register, handleSubmit: useHandleSubmit, control, setValue, getValues, reset, clearErrors, errors } = useForm<
+    Inputs
+  >({
+    defaultValues: {
+      avatar: channel.avatar || null,
+      cover: channel.cover || null,
+      channelName: channel.handle || '',
+      description: channel.description || '',
+      selectedLanguage: '',
+      selectedPublicness: '',
+    },
+  })
+  const avatarDialogRef = useRef<ImageCropDialogImperativeHandle>(null)
+  const coverDialogRef = useRef<ImageCropDialogImperativeHandle>(null)
+
+  const [avatarImageUrl, setAvatarImageUrl] = useState<string | null>(null)
+  const [coverImageUrl, setCoverImageUrl] = useState<string | null>(null)
+
+  const handleSubmit = useHandleSubmit((data) => {
+    console.log(data)
+  })
+  const handleAvatarConfirm = (blob: Blob, url: string) => {
+    setValue('avatar', url)
+    setAvatarImageUrl(url)
+  }
+  const handleCoverConfirm = (blob: Blob, url: string) => {
+    setValue('cover', url)
```

Comment:
I think we will need to save the blob as well, URL may not be enough to upload the file since we would need to read them again to get the data